### DISCUSSION TITLE
同步会失败且无法重新开始

### DISCUSSION THREAD

**Discussion by mrjeye** [messense/aliyundrive-webdav#375] [Q&A]
### 问题描述

备份过程中随机触发异常，且每首失败后，重新开始备份，**阿里云盘上所占用的空间不会被释放**，而是累加（**旧的已经备份过的数据还在阿里云盘上**）

### 重现步骤

1.登录群辉DSM
2.下载安装套件`阿里云盘Webdav`(`https://imnks.com/3939.html `)及群辉套件`Hyper Backup`
3.创建并开始备份任务
4.等待(时间不一定，可能在20%失败，可能在70%失败)备份任务失败
5.检查错误提示及日志
5.重新开启备份
6.观察

### 版本

1.3.1

### 运行平台

NAS(DS218+)

### 日志

1.`[WebDAV][ALIPAN_BACKUP] Exception occurred while backing up data. (The backup destination does not exist.)`
2.`[WebDAV][ALIPAN_BACKUP] Exception occurred while backing up data.`
3.`[WebDAV][ALIPAN_BACKUP] Exception occurred while backing up data. (The backup destination is busy rolling back because the last backup or version deletion failed. Please try again later.)`
4.`[WebDAV][ALIPAN_BACKUP] Exception occurred while backing up data. (The backup destination is busy rolling back because the last backup or version deletion failed. Please try again later.)`
5.`[WebDAV][ALIPAN_BACKUP] Backup task was suspended (suspension incomplete).`

**Comment by messense** [comment]
请提供本程序的日志而不是群辉程序的日志。
